src/spicy-vodka-pasta.md

```diff
@@ -0,0 +1,39 @@
+# Spicy Vodka Pasta
+
+A tasty and flavorful pasta.
+
+- ⏲️ Prep time: 10 min
+- 🍳Cook time: 20 min
+
```

Comment:
```suggestion
- 🍽️ Servings: 4

```

Reply:
It's actually 2 servings, let me fix it.